x-pack/plugins/maps/server/routes.js

```diff
@@ -78,6 +78,27 @@ export function initRoutes(server, licenseUid) {
     }
   });
 
+  server.route({
+    method: 'GET',
+    path: `${ROOT}/indexCount`,
+    handler: async (request, h) => {
+      const { server, query } = request;
+
+      if (!query.index) {
```

Comment:
Log a warning to the server-logs? This should never really occur, and if it does, it's unexpected and there is an issue with the stack deployment. Might be helpful to show feedback.

Reply:
This could happen if someone uses the API and does not supply `index` query param. Is it worth logging? Consumer of API would get 400 and know something is wrong with the request